lib/github.ml

```diff
@@ -178,8 +178,16 @@ let run_with_auth ?(default_body = `Null) ~dry_run ~auth curl_t =
     >>| fun () -> default_body
   else
     OS.Cmd.must_exist (Cmd.v "curl") >>= fun _ ->
-    match Curly.(run ~args (Request.make ~url ~meth:`POST ())) with
-    | Ok Curly.Response.{ body; _ } -> Json.from_string body
+    let req = Curly.Request.make ~url ~meth:`POST () in
+    Logs.debug (fun l ->
+        l "[curl] executing request:@;<1 2>%a" Curly.Request.pp req);
+    Logs.debug (fun l ->
+        l "[curl] with args:@;<1 2>%a" (Fmt.list Fmt.string) args);
```

Comment:
Might be good to set a `~sep` there don't you think? I'm not sure what it defaults to!

Reply:
indeed, done!